rs-backend/src/git.rs

```diff
@@ -0,0 +1,47 @@
+use std::env;
+use std::path::Path;
+use std::process::Command;
+
+/// Fetches repo path from supplied command line argument, panics otherwise.
+pub fn get_repo_path() -> String {
+    if env::args().count() < 2 {
+        panic!("No argument supplied, needs location of data repo.");
+    }
+    env::args().nth(1).unwrap()
+}
+
+
+pub fn update_repo(repo_path: String) {
+    let mut working_dir = Path::new(&repo_path);
+    if env::set_current_dir(working_dir).is_err() {
+        panic!("Invalid path specified");
+    }
+
+    let fetch = Command::new("git")
+        .arg("fetch")
+        .arg("origin")
+        .status()
+        .unwrap();
+
+    if fetch.success() {
+        let merge = Command::new("git")
+            .arg("merge")
+            .arg("master")
+            .arg("origin/master")
+            .status()
+            .unwrap();
+
+        if merge.success() {
+            let checkout = Command::new("git")
+                .arg("checkout")
+                .arg("head")
+                .arg("--force")
```

Comment:
Could you pull out some of these strings into `const`s please? It is probably worth factoring out the `Command` uses into a function too, since that seems to be repetitive code.